microcosm_pubsub/tests/chain/statements/test_call.py

```diff
@@ -0,0 +1,85 @@
+from hamcrest import (
+    assert_that,
+    calling,
+    equal_to,
+    raises,
+)
+
+from microcosm_pubsub.chain import Chain
+from microcosm_pubsub.chain.statements.call import call
+
+
+class TestLocalCallWrapper:
+    def test_two_calls_of_same_function(self):
+        def double_it(input):
+            return 2 * input
+
+        def multiply(double_one, double_three):
+            return double_one * double_three
+
+        chain = Chain(
+            call(double_it).with_args(input="one").as_("double_one"),
+            call(double_it).with_args(input="three").as_("double_three"),
```

Comment:
It's confusing that string literals pull from context but other literals don't.  I'd lean towards either not supporting literals or wrapping these strings in something.  Could be called `extract_from_context` or something